src/plugins/wizard/public/application/components/data_tab/dropbox.scss

```diff
@@ -13,24 +13,28 @@
 
   &__container {
     display: grid;
-    grid-gap: $euiSizeXS;
-    padding: $euiSizeS;
-    background-color: #e9edf3;
+    grid-gap: $euiSizeXS / 2;
```

Comment:
seems like a pain point worth noting as a painpoint in the design system for UX.

Reply:
Not really, the actual spacing here fits within the UX guidelines. The DOM structure of the dropbox is kinda tricky such that i need to add and subtract margins and paddings to make the UX look clean. The end result it still valid EUI space sizing